node.gyp

```diff
@@ -7,7 +7,6 @@
     'node_no_browser_globals%': 'false',
     'node_code_cache_path%': '',
     'node_use_v8_platform%': 'true',
-    'node_use_bundled_v8%': 'true',
```

Comment:
I'm wondering about this. node_use_bundled_v8 seems to be used in a few places and I'm wondering why removing is the right thing to do.

Reply:
It's duplicated in the more general `common.gpyi`: https://github.com/nodejs/node/blob/092ab7a1d39d97a3e93f8ed46d80d8ad783b2f9c/common.gypi#L19